Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,66 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Input",
+        "SQL"
+    ],
+    "name": "SqlInputBinding",
+    "identity": "Azure.Function.CSharp.SqlInputBinding",
+    "groupIdentity": "Azure.Function.SqlInputBinding",
+    "shortName": "sqlinput",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "SqlInputBindingCSharp",
+    "symbols": {
+        "connectionStringSetting": {
+            "description": "The name of the app setting containing your SQL connection string.",
+            "type": "parameter",
+            "defaultValue": "",
+            "replaces": "SqlConnectionString"
+        },
+        "object": {
+            "description": "This is the table or view name for the SQL input binding.",
+            "type": "parameter",
+            "defaultValue": "[dbo].[table1]",
+            "replaces": "object"
+        }
+    },
+    "primaryOutputs": [
+        {
+            "path": "SqlInputBindingCSharp.cs"
+        }
+    ],
+    "defaultName": "SqlInputBindingCSharp",
+    "postActions": [
+        {
+          "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.Sql Nuget package",
+          "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+          "ContinueOnError": "true",
+          "ManualInstructions": [],
+          "args": {
+            "referenceType": "package",
+            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "0.1.*-*",
```

Comment:
@soninaren - can you share why templates would be pegged to a specific library version?  this seems to me as an antipattern.
Wasting engineering resources to go through this template update change every time we release a new version is a terrible idea.

Reply:
@dzsquared You are right about the extra engineering effort required here. However that is not without reason. I have created this work item that would allow us to use a wild card in the extension version without negative impact. https://github.com/Azure/azure-functions-templates/issues/1153

Using the wild card `as is` does have some drawbacks which has impacted us before. I have copied the main concerns from the issue below. Let me know if you have questions.

1. The version resolution for the wild card characters is not done in a single library. Each dotnet host that consumes this template has its own custom code to resolve extension version from a wild card. To make sure if works as expected you would have to test each client individually. Even after they are tested there is a possibility of a bug being introduced or present after the fact. 
    - Visual studio
    - Dotnet CLI
    - Vs Code
    - VS for Mac
2. If there is bug that is introduced in any of the extension releases, customers would automatically run into it without updating any tools / environment. They would have no way to know what caused the issue. This would particularly cause issues in the automated test environments.
3. Additionally that makes the tooling unreliable for the customers and the journey from a broken state to a fix would be very long and tedious for the customer, not to mention the servicing / debugging effort required by the team.

Reply:
thanks for elaborating @soninaren and appreciate the proactively logged issue. we'll get this set to a static value so we can move forward and look forward to the improvement